client/src/components/navigation/navConfig.tsx

```diff
@@ -0,0 +1,126 @@
+import { Dashboard, PeopleAlt, Event, Build, MoreHoriz, EmojiObjects, TrendingUp } from "@mui/icons-material";
+import { SvgIconComponent } from "@mui/icons-material";
+import { Permission } from "../../types/globalTypes";
+
+interface NavItem {
+  title: string;
+  path?: string;
+  icon: SvgIconComponent;
+  children?: {
+    title: string;
+    path: string;
+    permissions?: Permission[];
+  }[];
+}
+
+export function getIconByName(name: string) {
```

Comment:
We use the arrow function format for functions (() => {...}).


```suggestion
export const getIconByName = (name: string) => {
```